src/authentication.ts

```diff
@@ -88,7 +100,8 @@ export async function authenticate(options?: AuthenticationOptions): Promise<Cre
       return {
         port: Number(port),
         pid: Number(pid),
-        password
+        password,
+        certificate: options?.certificate || (await fs.readFile(path.join(__dirname, '..', 'riotgames.pem'), 'utf8'))
```

Comment:
We should probably not take the assumption that the pem certificate is in `../riotgames.pem` as users/applications may have very different structures. The file location would most likely be incorrect when the module is installed as it'd point to node_modules/league-connect when it should probably be in the user's project directory.